### PR TITLE
Update nspanel_esphome.yaml

### PR DESCRIPTION
**Pull request by danielmessem**

### CODE REVIEW

File: nspanel_esphome.yaml [hunk 1]
```diff
@@ -361,7 +361,7 @@ globals:
   - id: temperature_correction_global
     type: float
     restore_value: true
-    initial_value: '0.0'
+    initial_value: '-1.2'
```
Comment (edwardtfn) [inline nspanel_esphome.yaml:364]: Could you please explain why having `-1.2` and not `0.0` as initial correction for temperature?

File: nspanel_esphome.yaml [hunk 2]
```diff
@@ -936,9 +936,9 @@ number:
     id: temperature_correction
     entity_category: config
     unit_of_measurement: '°C'
-    min_value: -10
-    max_value: 10
-    step: 0.5
+    min_value: -3
```
Comment (edwardtfn) [inline nspanel_esphome.yaml:939]: Don't you thing the temperature will deviate more than this new range you are proposing?

**Comment by edwardtfn** [comment]
@danielmessem, could you please add more details on what you are trying to solve with this and how you are trying to solve?

Also, please use DEV as base branch for PRs.

File: nspanel_esphome.yaml (hunk 1 above)
Comment (danielmessem) [reply nspanel_esphome.yaml:364]: Really sorry please reject, 1st time using github, this was only a change I was playing around with for my device

**Comment by edwardtfn** [comment]
No need for apologies, we all have started with GitHub one day... And I can say I still a beginner here. 😉

When you created your pull request with the change, GitHub automatically created a fork of this project for you under `danielmessem/NSPanel_HA_Blueprint`.
You can use that fork in your ESPHome settings by using your name replacing the original one in the `url`:
IMG_6178

As you have created a branch when creating your pull requests with your changes, you probably still have to merge this changes to your own main branch or maybe you can just change `ref` in your ESPHome configuration to point it to the right direction.

But if you find something that makes your solution better, please propose a PR here just like you have done (but to DEV), so everyone can take advantage of your journey. 😉

**Comment by deejaybeam** [comment]
no problem - thanks for your clarification.

**Pull request closed without merging**